### PR TITLE
fix theos bug decontainment unit

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

can only store 51 * level items
can't be using as item storage while unanchored
# Spriting

# Wiki Documentation

# Changelog

:cl:  
bugfix: You can no longer use the decontamination unit as a item storage while unanchored, and it only stores 51 * level items now.
/:cl:

### CODE REVIEW

File: code/game/machinery/decontamination.dm [hunk 1]
```diff
@@ -117,7 +117,7 @@
 			flick("tube_up", src)
 			decon_emagged.stop()
 			playsound(src, 'sound/machines/decon/decon-up.ogg', 100, TRUE)
-			sleep(8)
+			sleep(8) //stop
```
Comment (ShoziOffical) [inline code/game/machinery/decontamination.dm]: this is to wait for the "tube_up" to finish its animation before opening the door

File: code/game/machinery/decontamination.dm [hunk 2]
```diff
@@ -132,7 +132,7 @@
 			flick("tube_up", src)
 			decon.stop()
 			playsound(src, 'sound/machines/decon/decon-up.ogg', 100, TRUE)
-			sleep(8)
+			sleep(8) //why
```
Comment (ShoziOffical) [inline code/game/machinery/decontamination.dm]: this is to wait for the "tube_up" to finish its animation before opening the door

Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm]: also can you increase it by 0.5

File: code/game/machinery/decontamination.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm]: also can you increase it by 0.5

**Review by warface1234455 — CHANGES_REQUESTED** [review]
please mr simplehorror

File: code/game/machinery/decontamination.dm (hunk 1 above)
Comment (Simplehorror) [reply code/game/machinery/decontamination.dm]: we shouldn't be using sleep anymore

File: code/game/machinery/decontamination.dm (hunk 2 above)
Comment (Simplehorror) [reply code/game/machinery/decontamination.dm]: we shouldn't be using sleep anymore

**Comment by Simplehorror** [comment]
no way jose do it yourself im not touching sleeps

**Comment by ShoziOffical** [comment]
> no way jose do it yourself im not touching sleeps

fine

File: code/game/machinery/decontamination.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm]: give me something better

File: code/game/machinery/decontamination.dm (hunk 2 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm]: give me something better

**Review by MajManatee — CHANGES_REQUESTED** [review]
Don't use sleep, use addtimer instead.

**Comment by ShoziOffical** [comment]
> Don't use sleep, use addtimer instead.

That is on another pr

**Review by MajManatee — APPROVED** [review]
it really was fixed in another PR

**Comment by MajManatee** [comment]
whats to stop people from just... wrenching it down, opening, storing items, closing and unwrenching?

**Comment by Moltijoe** [comment]
> whats to stop people from just... wrenching it down, opening, storing items, closing and unwrenching?

tedium i guess?

**Pull request merged**